### PR TITLE
Fix trivial log issue

### PR DESCRIPTION
**Pull request by kangkaisen**
Currently,  `explain insert into t1 SelectStmt` output is empty.

After fixing this bug, `explain insert into t1 select * from t2;` output is following:

```
+----------------------------------------------------------------------------------------------------------------------+
| Explain String                                                                                                       |
+----------------------------------------------------------------------------------------------------------------------+
| PLAN FRAGMENT 0                                                                                                      |
|  OUTPUT EXPRS:`default_cluster:test.t2`.`dt` | `default_cluster:test.t2`.`id` | `default_cluster:test.t2`.`value` |  |
|   PARTITION: RANGE_PARTITIONED: <slot 0>                                                                             |
|                                                                                                                      |
|   DPP SINK                                                                                                           |
|     TABLE NAME: t1                                                                                                   |
|     PARTITIONS: 10008, 10009, 10010                                                                                  |
|                                                                                                                      |
|   2:EXCHANGE                                                                                                         |
|      tuple ids: 0                                                                                                    |
|                                                                                                                      |
| PLAN FRAGMENT 1                                                                                                      |
|  OUTPUT EXPRS:                                                                                                       |
|   PARTITION: RANDOM                                                                                                  |
|                                                                                                                      |
|   STREAM DATA SINK                                                                                                   |
|     EXCHANGE ID: 02                                                                                                  |
|     RANGE_PARTITIONED: <slot 0>                                                                                      |
|                                                                                                                      |
|   1:OLAP REWRITE NODE                                                                                                |
|   |  tuple ids: 0                                                                                                    |
|   |                                                                                                                  |
|   0:OlapScanNode                                                                                                     |
|      TABLE: t2                                                                                                       |
|      PREAGGREGATION: ON                                                                                              |
|      partitions=1/1                                                                                                  |
|      rollup: t2                                                                                                      |
|      buckets=10/10                                                                                                   |
|      cardinality=2                                                                                                   |
|      avgRowSize=1889.4286                                                                                            |
|      tuple ids: 1                                                                                                    |
+----------------------------------------------------------------------------------------------------------------------+
```

### CODE REVIEW

**Comment by kangkaisen** [comment]
Update this commit.

File: fe/src/main/java/org/apache/doris/analysis/InsertStmt.java [hunk 1]
```diff
@@ -233,7 +233,7 @@ public void analyze(Analyzer analyzer) throws UserException {
                 sink.init(loadId, transactionId, db.getId());
             }
         }
-        LOG.info("analyzer is ", analyzer.getDescTbl().debugString());
+        LOG.info("analyzer is {}", analyzer.getDescTbl().debugString());
```
Comment (imay) [inline fe/src/main/java/org/apache/doris/analysis/InsertStmt.java]: Why changing a log can make explain command work?

Comment (kangkaisen) [reply fe/src/main/java/org/apache/doris/analysis/InsertStmt.java]: I'm sorry，It's my negligence.

The InsertStmt explain issue has been fixed.

Comment (imay) [reply fe/src/main/java/org/apache/doris/analysis/InsertStmt.java]: I think this is a debug log, just delete it?

Comment (kangkaisen) [reply fe/src/main/java/org/apache/doris/analysis/InsertStmt.java]: OK.

**Review by imay — APPROVED** [review]

**Pull request merged**